vm/systemSmartContracts/delegation.go

```diff
@@ -64,6 +64,8 @@ type delegation struct {
 	validatorToDelegationEnableEpoch   uint32
 	flagReDelegateBelowMinCheck        atomic.Flag
 	reDelegateBelowMinCheckEnableEpoch uint32
+	liquidStakingEnableEpoch           uint32
```

Comment:
undefined epoch in enableEpochs.toml + print in the /node/nodeRunner.go

Reply:
BuiltInFunctionOnMetaEnableEpoch is used for this